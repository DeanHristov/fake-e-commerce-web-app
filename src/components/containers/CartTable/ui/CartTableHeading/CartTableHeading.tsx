import { FC } from 'react';

export interface ICartTableHeadingProps {}

const CartTableHeading: FC<ICartTableHeadingProps> = () => {
  return (
    <ul className="w-full flex border-b border-t border-b-dark border-t-dark">
      <li className="w-full px-2 py-2 text-center md:text-left">
        <span className="">Item</span>
      </li>
      <li className="w-full max-w-[8rem] px-2 py-2 hidden md:block">
        <span className="">Price</span>
      </li>
      <li className="w-full max-w-[8rem] px-2 py-2 hidden md:block">
        <span className="">Quantity</span>
      </li>
      <li className="w-full max-w-[10rem] px-2 py-2 hidden md:block">
        <span className="">Subtotal</span>
      </li>
    </ul>
  );
};

export default CartTableHeading;
