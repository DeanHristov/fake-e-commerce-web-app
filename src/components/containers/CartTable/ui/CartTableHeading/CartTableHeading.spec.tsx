import { render } from '@testing-library/react';

import CartTableHeading from './CartTableHeading';

describe('UI/CartTableHeading <CartTableHeading {...} />', () => {
  it('Should render a default state of the Component', () => {
    const { container } = render(<CartTableHeading />);

    expect(container).toMatchSnapshot();
  });
});
