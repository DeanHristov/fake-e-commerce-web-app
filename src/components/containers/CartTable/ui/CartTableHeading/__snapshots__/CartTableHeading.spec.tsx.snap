// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/CartTableHeading <CartTableHeading {...} /> Should render a default state of the Component 1`] = `
<div>
  <ul
    class="w-full flex border-b border-t border-b-dark border-t-dark"
  >
    <li
      class="w-full px-2 py-2 text-center md:text-left"
    >
      <span
        class=""
      >
        Item
      </span>
    </li>
    <li
      class="w-full max-w-[8rem] px-2 py-2 hidden md:block"
    >
      <span
        class=""
      >
        Price
      </span>
    </li>
    <li
      class="w-full max-w-[8rem] px-2 py-2 hidden md:block"
    >
      <span
        class=""
      >
        Quantity
      </span>
    </li>
    <li
      class="w-full max-w-[12rem] px-2 py-2 hidden md:block"
    >
      <span
        class=""
      >
        Subtotal
      </span>
    </li>
  </ul>
</div>
`;
